site/2016/10/07/doctrine-mongodb-odm-release-1.1.2-and-1.0.8.rst

```diff
@@ -0,0 +1,50 @@
+Doctrine MongoDB ODM 1.1.2 and 1.0.8 released
+=============================================
+
+We are happy to announce the immediate availability of Doctrine MongoDB ODM
+`1.1.2 <https://github.com/doctrine/mongodb-odm/releases/tag/1.1.2>`__ and
+`1.0.8 <https://github.com/doctrine/mongodb-odm/releases/tag/1.0.8>`__.
+
+MongoDB ODM 1.0.8
+-----------------
+
+- calling ``dropCollections()`` in SchemaManager did not drop GridFS collections.
+This has been fixed in `#1468 <https://github.com/doctrine/mongodb-odm/pull/1468>`_.
```

Comment:
I wouldn't use `This has been fixed in` for every entry. Just link it with the issue ID